projects/plugins/jetpack/extensions/blocks/recipe/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ExternalLink } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import attributes from './attributes';
+import edit from './edit';
+import save from './save';
+import icon from './icon';
+import { getIconColor } from '../../shared/block-icons';
+import { name as detailsName, settings as detailsSettings } from './details/';
+import { name as heroName, settings as heroSettings } from './hero/';
+import {
+	name as ingredientsListName,
+	settings as ingredientsListSettings,
+} from './ingredients-list/';
+import { name as ingredientItemName, settings as ingredientItemSettings } from './ingredient-item/';
+import { name as stepsName, settings as stepsSettings } from './steps/';
+import { name as stepName, settings as stepSettings } from './step/';
+
+export const name = 'recipe';
+export const title = __( 'Recipe', 'jetpack' );
+export const settings = {
+	title,
+	description: (
+		<Fragment>
+			<p>
+				{ __(
+					'Add images, ingredients and cooking steps to display an easy to read recipe.',
+					'jetpack'
+				) }
+			</p>
+			<ExternalLink href="#">{ __( 'Learn more about Recipe', 'jetpack' ) }</ExternalLink>
+		</Fragment>
+	),
+	icon: {
+		src: icon,
+		foreground: getIconColor(),
+	},
+	category: 'jetpack',
+	keywords: [],
```

Comment:
I think we can start adding a few keywords in there, related to food and recipes.